server/lib/validation/languages/am.js

```diff
@@ -0,0 +1,32 @@
+const tokenizeWords = require('talisman/tokenizers/words');
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 6;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 26;
```

Comment:
Thanks for contributing this validation file! This generally looks good to me, though I have two questions:

* How long does it take on average roughly to speak out aloud 6 words?
* How long for 26?

Thanks!

Reply:
@MichaelKohler Thank you for your fast reply, and you asked me a very important question. to answer your questions.

- On average to speak out aloud 6 words takes 5 seconds, and
- On average speaking out aloud 26 words takes 15 seconds.
Thank you.

Reply:
Thanks. In that case I would probably suggest 3-4 words minimum if reasonable sentences of that size exist.

For the maximum we have a limit on the Common Voice platform of 10 seconds for recordings, so I would suggest something around 8 seconds maximum to make sure that people speaking a bit slower won't get many of their contributions cut off.

Reply:
Oh if that is the case more than 16 words will exceed the maximum 10-second recording I guess. So, how about we make it 3 minimum and 15 maximum words?

Reply:
Yes, I think that sounds reasonable to do. Thanks!

Reply:
OK. let me change the file then.